.ci/scripts/run_sklearn_tests.py

```diff
@@ -37,22 +37,38 @@
     )
     args = parser.parse_args()
 
-    os.chdir(os.path.dirname(sklearn.__file__))
+    cwd = os.getcwd()
+    root_dir = os.path.dirname(sklearn.__file__)
+    os.chdir(root_dir)
 
     if os.environ["SELECTED_TESTS"] == "all":
         os.environ["SELECTED_TESTS"] = ""
 
     pytest_args = (
         "--verbose --durations=100 --durations-min=0.01 "
-        f"--rootdir={os.path.dirname(sklearn.__file__)} "
+        f"--rootdir={root_dir} "
         f'{os.environ["DESELECTED_TESTS"]} {os.environ["SELECTED_TESTS"]}'.split(" ")
     )
+
+    if os.getenv("COVERAGE_RCFILE"):
+        pytest_args += (
+            f'--cov=onedal --cov=sklearnex --cov-config={os.environ["COVERAGE_RCFILE"]} '
+            "--cov-report=term".split(" ")
+        )
+    print("to be run: pytest " + " ".join(pytest_args))
+
     while "" in pytest_args:
         pytest_args.remove("")
 
     if args.device != "none":
         with sklearn.config_context(target_offload=args.device):
-            return_code = pytest.main(pytest_args)

```

Comment:
Why was the conversion added? What were non-int returns previously?

Reply:
Conversion to int occurs at the end of the file, I wanted to have it earlier to check for a successful run, so I moved it into the return code.